### ISSUE TITLE
Integrate nautilus extension into Cast to TV

### ISSUE THREAD

**Issue by Rafostar** [rendyanthony/nautilus-cast#3]
Hi,
I really like your work on this nautilus plugin. I never did nautilus python extension before and your code helps me a lot. Thanks!

Would it be OK with you if I did my own implementation of nautilus extension with added few missing features using some portions of your code and bundle it with my gnome extension? I will of course give you proper credit on my extension readme page. It would allow users to have this great feature right after extension installation.

Edit: And because your extension differs from what I did recently I would also add link to your code, so users will be pointed to yours implementation if they prefer it.

**Comment by rendyanthony** [comment]
Hi,

Thanks you very much.  This is also my first time doing a Nautilus Extension myself. I also really like your Gnome Shell extension, it's one of the cleanest way for me to cast local media files to the TV.

I'm fine with you implementing your own Nautilus extension as long as you link back to my repo. It would be great for the users to have the feature in one installation.

**Comment by Rafostar** [comment]
> It would be great for the users to have the feature in one installation.

My thoughts exactly. Thank you very much!

By the way I noticed that you use `devices.json` file from extension folder. In next release this file will be located inside "config" folder in extension dir (`config/devices.json`). I changed this few weeks ago when I was adding subtitles customisation to extension prefs. You might need to change this path in your code for next version or add some kind of detection where this file is (two possibilities). Sorry for confusion.
